arch/arm64/boot/dts/qcom/sdm450-motorola-ali.dts

```diff
@@ -0,0 +1,291 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright (c) 2020, The Linux Foundation. All rights reserved.
+
+/dts-v1/;
+
+#include "msm8953.dtsi"
+#include "pm8953.dtsi"
+#include "pmi8950.dtsi"
+
+/ {
+  model = "ali";
+  compatible = "motorola,ali", "qcom,sdm450";
+  qcom,msm-id = <338 0>;
+  qcom,board-id = <0x41 0xb1a0 0x42 0xb1a0>, <0x43 0xc100>, <0x42 0xb1b0>, <0x43 0xc200 0x44 0xc200>, <0x42 0xb200 0x43 0xb200>;
+
+  reserved-memory {
+  	/delete-node/other_ext_region@85b00000;
+
+  	other_ext_region@84300000 {
+  		no-map;
+  		reg = <0x0 0x84300000 0x0 0x2000000>;
+  	};
+
+  	ramoops_mem: ramoops_mem_region {
+  		no-map;
+  		compatible = "ramoops";
+  		reg = <0x0 0xef000000 0x0 0xC0000>;
+  		console-size = <0x40000>;
+  	};
+
+  };
+
+  vph_pwr: vph-pwr-regulator {
+  	compatible = "regulator-fixed";
+  	regulator-name = "vph_pwr";
+  	regulator-always-on;
+  	regulator-boot-on;
+  };
+
+  gpio-keys {
+  	compatible = "gpio-keys";
+
+  	volume-up {
+  		label = "volume_up";
+  		linux,code = <KEY_VOLUMEUP>;
+  		gpios = <&tlmm 85 GPIO_ACTIVE_LOW>;
+  	};
+  };
+};
+
+&i2c_3 {
+	status = "okay";
+
+	touchscreen@38 {
+		compatible = "edt,edt-ft5406";
+		reg = <0x38>;
+		interrupt-parent = <&tlmm>;
+		interrupts = <65 IRQ_TYPE_EDGE_FALLING>;
+		vcc-supply = <&pm8953_l10>;
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&ts_int_active &ts_reset_active>;
+
+		reset-gpios = <&tlmm 64 GPIO_ACTIVE_LOW>;
+		touchscreen-size-x = <1080>;
+		touchscreen-size-y = <2160>;
+	};
+};
+
+&pmi8950_wled {
+	status = "okay";
+
+	/* Don't copy-paste! See pmi8950.dtsi */
+
+	qcom,current-limit-microamp = <25000>;
+	qcom,enabled-strings = <0 1 2>;
+	qcom,num-strings = <3>;
+
+	qcom,cabc;
+	qcom,eternal-pfet;
```

Comment:
it's external, not eternal as reported in 

Documentation/devicetree/bindings/leds/backlight/qcom-wled.yaml

and in

drivers/video/backlight/qcom-wled.c